sdk/samples/iot/README.md

```diff
@@ -176,13 +173,53 @@ To run the samples, ensure you have the following programs and tools installed o
     git clone https://github.com/Azure/azure-sdk-for-c.git
     ```
 
+- If running a DPS sample: `paho_iot_provisioning_sample`, `paho_iot_provisioning_sas_sample`
+  - Have an [Azure IoT Hub Device Provisioning Service (DPS)](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision) created.
+
+- If running a Plug and Play sample: `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`
+  - Have the most recent version of [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases) installed and connected to your Azure IoT Hub. More instructions on can be found [here](https://docs.microsoft.com/azure/iot-pnp/howto-use-iot-explorer).
+
+- If running a CBOR sample: `paho_iot_hub_twin_cbor_sample`
+  - Have the [MIT licensed](https://github.com/intel/tinycbor/blob/master/LICENSE) intel/tinycbor library installed.
+    <details><summary><i>Instructions:</i></summary>
+    <p>
+
+    Linux:
+
+    ```
+    git clone https://github.com/intel/tinycbor.git
+    cd tinycbor
+    git checkout v0.5.3
+    make
+    sudo make install
+    ```
+
+    - Please check your `/usr/local/lib` directory and update permissions on the installed libraries if needed.
+
+    Windows (Select the Command Prompt):
```

Comment:
nit: Is there a way to add some line break/spacing here?
Looks squished in the markdown:
![image](https://user-images.githubusercontent.com/6527137/115097302-19571d80-9ede-11eb-9886-4b549eca2003.png)

Reply:
good eye